contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -590,6 +590,53 @@ pltsql_pre_parse_analyze(ParseState *pstate, RawStmt *parseTree)
 		}
 	}
 
+	if (parseTree->stmt->type == T_AlterTableStmt)
+	{
+		AlterTableStmt *atstmt = (AlterTableStmt *) parseTree->stmt;
+		ListCell *lc;
+		char *trig_schema;
+		const char *rel_schema;
+
+		foreach(lc, atstmt->cmds)
+		{
+			AlterTableCmd *cmd = (AlterTableCmd *)lfirst(lc);
+			if (cmd->subtype == AT_EnableTrig || cmd->subtype == AT_DisableTrig)
+			{
+				if (strchr(cmd->name, '.') != NULL)
+				{
+					trig_schema = strtok(cmd->name, ".");
+					cmd->name = strtok(NULL, ".");
+				}
+				else
+				{
+					trig_schema = NULL;
+				}
+				
+				if (atstmt->relation->schemaname != NULL)
+				{
+					rel_schema = get_logical_schema_name(atstmt->relation->schemaname, true);
+
+					if (rel_schema == NULL)
+					{
+						rel_schema = atstmt->relation->schemaname;
+					}
+				}
+				else
+				{
+					rel_schema = "dbo";
```

Comment:
Default schema can be change by the user, Instead of hard-coding it to `dbo` we should lookup for the default schema of the current user. Please check with T-SQL, update here accordingly.

Reply:
Updated